### ISSUE TITLE
[10.0.0] readOnly cells are hardly visible

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#8770]
### Description

v10 introduced changes to the default appearance, including a font color change to `#373737`.
The result is that the cells marked as `readOnly` (`htDimmed`) with the `#777` color have much lower contrast and are less visible compared to a normal color font `#373737`.
Such a small contrast between the background and the font does not pass [accesibillity tests](https://webaim.org/resources/linkcontrastchecker/?fcolor=777777&bcolor=FFFFFF&lcolor=373737).

### Expected solution

In the course of discussions with the team, @krzysztofspilka recommended that the readOnly cells should have the background color `#E9E9E9` which is used as highlight items in the context menu.

**v10**
![now](https://user-images.githubusercontent.com/30186818/135218381-002bec1f-56bd-4e3b-b3d5-d61c5799264a.gif)

**v9**
![then](https://user-images.githubusercontent.com/30186818/135218408-46c3d94c-f228-4da2-9d04-6f0793a57ff4.png)

### Demo

https://jsfiddle.net/d2azc105/1/ v10.0.0
https://jsfiddle.net/d2azc105/2/ v9.0.2

### Your environment
* Handsontable version: 10.0.0
* Browser Name and version: any
* Operating System: any
